### ISSUE TITLE
Server returns empty string for major and minor version

### ISSUE THREAD

**Issue by aaron7** [kubernetes/minikube#2505]
### Bug report

**Minikube version**:  0.25.0
- **OS**: OSX High Sierra
- **VM Driver**: virtualbox
- **ISO version**: minikube-v0.25.1.iso
- **Install tools**: brew cask
- **Others**:

**What happened**:

Server is returning major and minor versions as empty strings.

```
❯❯❯ kubectl version
Client Version: version.Info{Major:"1", Minor:"9", GitVersion:"v1.9.0", GitCommit:"925c127ec6b946659ad0fd596fa959be43f0cc05", GitTreeState:"clean", BuildDate:"2017-12-16T03:16:50Z", GoVersion:"go1.9.2", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"", Minor:"", GitVersion:"v1.9.0", GitCommit:"925c127ec6b946659ad0fd596fa959be43f0cc05", GitTreeState:"clean", BuildDate:"2018-01-26T19:04:38Z", GoVersion:"go1.9.1", Compiler:"gc", Platform:"linux/amd64"}
```

Server Version: `Major:"", Minor:""`

**What you expected to happen**:

Server Version: `Major:"1", Minor:"9"`

**How to reproduce it** (as minimally and precisely as possible):

`kubectl version`

**Output of `minikube logs` (if applicable)**:

**Anything else do we need to know**:

**Comment by afbjorklund** [comment]
This looks like an upstream change in kubernetes 1.9, where they were moved to variables:

https://github.com/kubernetes/minikube/blob/28cbc6f73afd0e7c41bd7fde54888f8d381faf3b/vendor/k8s.io/client-go/pkg/version/base.go#L37-L43

So they now need to be passed by the build system, and "localkube" isn't doing that properly:

https://github.com/kubernetes/kubernetes/blob/master/pkg/version/def.bzl#L11
https://github.com/kubernetes/kubernetes/blob/master/hack/lib/version.sh#L144

https://github.com/kubernetes/minikube/blob/28cbc6f73afd0e7c41bd7fde54888f8d381faf3b/hack/get_k8s_version.py#L54-L59

**Comment by afbjorklund** [comment]
Fix in ab52ab1f81b2e7f72caa78b55047cbe018b6a33c, but localkube-v1.9.0 needs to be rebuilt...
